### ISSUE TITLE
Sprockets::Railtie::ManifestNeededError during instalation

### ISSUE THREAD

**Issue by Yanusz** [docker-library/redmine#176]
A default docker  instalation

`$ docker run -d --name some-redmine redmine`

 fails with the following message:

```
The Gemfile's dependencies are satisfied
rake aborted!
Sprockets::Railtie::ManifestNeededError: Expected to find a manifest file in `app/assets/config/manifest.js`
But did not, please create this file and use it to link any assets that need
to be rendered by your app:

Example:
  //= link_tree ../images
  //= link_directory ../javascripts .js
  //= link_directory ../stylesheets .css
and restart your server
```

Steps to reproduce the error:
```

[node2] (local) root@192.168.0.32 ~
$ docker --version
Docker version 19.03.1, build 74b1e89e8a
[node2] (local) root@192.168.0.32 ~
$ docker run -d --name some-redmine redmine
Unable to find image 'redmine:latest' locally
latest: Pulling from library/redmine
8d691f585fa8: Pull complete 
50f5caf807b8: Pull complete 
2cf92e4c3b7e: Pull complete 
d0df3e7ef5c8: Pull complete 
8362bb79af85: Pull complete 
da3200d12b6d: Pull complete 
ba18b97aa120: Pull complete 
49a5a62262bf: Pull complete 
266856f57a73: Pull complete 
ea586cba86b7: Pull complete 
e778ed82421e: Pull complete 
e95e075f2816: Pull complete 
b54c3a0ff852: Pull complete 
Digest: sha256:8eb37d401246dbda76d6d6f9a6ed452c1e90e0251a1247a1ef0f74001b3507df
Status: Downloaded newer image for redmine:latest
dc5efe4d6cc9ac06e12e61ccdd0682932d5a4d3372fcfd49b57710a15d091df0
[node2] (local) root@192.168.0.32 ~
$ docker ps -a
CONTAINER ID        IMAGE               COMMAND                  CREATED             STATUS                    PORTS               NAMES
dc5efe4d6cc9        redmine             "/docker-entrypoint.…"   7 seconds ago       Exited (1) 1 second ago                       some-redmine
[node2] (local) root@192.168.0.32 ~
$ docker logs some-redmine

warning: missing REDMINE_DB_MYSQL, REDMINE_DB_POSTGRES, or REDMINE_DB_SQLSERVER environment variables

*** Using sqlite3 as fallback. ***

The dependency tzinfo-data (>= 0) will be unused by any of the platforms Bundler is installing for. Bundler is installing for ruby but the dependency is only for x86-mingw32, x64-mingw32, x86-mswin32. To add those platforms to the bundle, run `bundle lock --add-platform x86-mingw32 x64-mingw32 x86-mswin32`.
The Gemfile's dependencies are satisfied
rake aborted!
Sprockets::Railtie::ManifestNeededError: Expected to find a manifest file in `app/assets/config/manifest.js`
But did not, please create this file and use it to link any assets that need
to be rendered by your app:

Example:
  //= link_tree ../images
  //= link_directory ../javascripts .js
  //= link_directory ../stylesheets .css
and restart your server
/usr/local/bundle/gems/sprockets-rails-3.2.1/lib/sprockets/railtie.rb:105:in `block in <class:Railtie>'
/usr/local/bundle/gems/railties-5.2.3/lib/rails/initializable.rb:32:in `instance_exec'
/usr/local/bundle/gems/railties-5.2.3/lib/rails/initializable.rb:32:in `run'
/usr/local/bundle/gems/railties-5.2.3/lib/rails/initializable.rb:61:in `block in run_initializers'
/usr/local/bundle/gems/railties-5.2.3/lib/rails/initializable.rb:60:in `run_initializers'
/usr/local/bundle/gems/railties-5.2.3/lib/rails/application.rb:361:in `initialize!'
/usr/src/redmine/config/environment.rb:14:in `<top (required)>'
/usr/local/bundle/gems/railties-5.2.3/lib/rails/application.rb:337:in `require'
/usr/local/bundle/gems/railties-5.2.3/lib/rails/application.rb:337:in `require_environment!'
/usr/local/bundle/gems/railties-5.2.3/lib/rails/application.rb:520:in `block in run_tasks_blocks'
/usr/local/bundle/gems/rake-13.0.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => db:migrate => db:load_config => environment
(See full trace by running task with --trace)
[node2] (local) root@192.168.0.32 ~
```

**Comment by vzvu3k6k** [comment]
ref. http://www.redmine.org/issues/32223

This issue has been solved on Redmine 4.0.5 and 3.4.12. These images are not pushed yet, but you can build images by yourself as `Dockerfile`s for the latest version has been committed (4064a505db909322bda24823c200f4c97985d638, ba9880cc3a224c7e5e15c1f9bf1757aa7c4c5b92).

**Comment by Yanusz** [comment]
Ok, thanks. It worked :)
